create security.md for new Github security feature (#435)
PrivateBin/PrivateBin: SECURITY.md

# Security Policy

## Supported Versions

| Version | Supported          |
| ------- | ------------------ |
| 1.2.1   | :white_check_mark: |
| < 1.2.1 | :x:                |

## Reporting a Vulnerability

> We run a mailing list under security@privatebin.org. You should be able to get

Reviewer: ```suggestion
You can send us a mail at security@privatebin.org. You should be able to get
```

…as it's not really a mailing list.

Author: It is a mail distribution list, not a full email account. Senders can't therefore not expect to get a reply from this address.
